serializers.py

```diff
@@ -0,0 +1,13 @@
+def director_from_web(**kwargs):
+    return\
+        {
+            "id": kwargs["id"] if kwargs["id"] else "",
+            "nome_completo": kwargs["nome_completo"] if kwargs["nome_completo"] else "",
+        }
+
+def director_from_db(*args):
+    return\
+        {
+            "id": args[0],
+            "nome_completo": args[1]
+        }
```

Comment:
Como a gente mudou o tipo de retorno, agora esse aqui fica assim
```suggestion
def director_from_db(args):
    return {
            "id": args["id"],
            "nome_completo": args["nome_completo"]
        }
```